Simplified/Reader2/UI/NYPLBaseReaderViewController.swift

```diff
@@ -192,22 +188,6 @@ class NYPLBaseReaderViewController: UIViewController, NYPLBackgroundWorkOwner, L
     return navigationBarHidden && !UIAccessibility.isVoiceOverRunning
   }
 
-  //----------------------------------------------------------------------------
-  // MARK: - NYPLBackgroundWorkOwner
-  // TODO: SIMPLY-2804
-
-  func setUpWorkItem(wrapping backgroundWork: @escaping () -> Void) -> (() -> Void)? {
-    return { }
-  }
-
-  func performBackgroundWork() {
-//    let bookMapDict: [AnyHashable : Any] = [:]//NYPLReaderReadiumView.generateBookDictionary
```

Comment:
I'm fine to remove this background work (I never quite understood why we had it, I'm guessing in the past it took so long that the user would background the app) but did you verify this work is not taking a really long time? It didn't in my tests in R1 but just wanted to make sure. If you see any unusual delays let's discuss this.

Reply:
From my understanding, we were sending a synchronous network request to retrieve the spine element and TOC information. That's why it is expensive.
I don't think we need it for R2.